refactor: Use constants on some preferences keys

BrayanDSO opened the pull request:
## Pull Request template

## Purpose / Description
I wanted more safety on dealing with some preferences keys

## Fixes
Helps #10423

## Approach
Add constants to `gestures` and `gestureCornerTouch` and add some uses of the existing `timeoutAnswer` constant

## How Has This Been Tested?

Lint passes

## Checklist
_Please, go through these checks before submitting the PR._

- [X] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [X] You have a descriptive commit message with a short title (first line, max 50 chars).
- [X] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [X] You have commented your code, particularly in hard-to-understand areas
- [X] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

david-allison commented on AnkiDroid/src/main/res/values/preferences.xml:
```diff
@@ -26,4 +26,6 @@
     <string name="app_theme_key">appTheme</string>
     <string name="day_theme_key">dayTheme</string>
     <string name="night_theme_key">nightTheme</string>
+    <string name="gestures_preference" comment="key for the gestures preference">gestures</string>
```
@BrayanDSO, would you mind if I blocked this PR on a 3-PR chain starting with #11037 - you shouldn't need a comment on `preferences.xml` and I'd rather fix this before it gets forgotten

BrayanDSO replied:
Go for it 👍. The idea behind the PRs looks great.

mikehardy replied:
I did my best to get the stack of PRs that this depends on to the head of the queue and I think they're all in. Not sure what the plans are here but I *think* we're all unblocked. Give me a holler if I can expedite or anything

david-allison replied:
One more to go

david-allison replied:
Unblocked: #11070

mikehardy replied:
That was quite the PR chain :-)

BrayanDSO commented:
Rebased into main and removed the comments. On `gradlew lint`:
```
> Task :AnkiDroid:lintAmazonDebug FAILED
C:\Users\Brayan\Dev\Anki-Android\AnkiDroid\src\main\res\values\10-preferences.xml:35: Error: Duplicate string value "gestures", used in gestures_preference, pref_cat_gestures. Use android:inputType or android:capitalize to treat these as the same and avoid string duplication. [DuplicateCrowdInStrings f
rom com.ichi2.anki:lint-rules]
    <string name="pref_cat_gestures" maxLength="41">Gestures</string>
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    C:\Users\Brayan\Dev\Anki-Android\AnkiDroid\src\main\res\values\preferences.xml:29: Duplicates value in pref_cat_gestures. Add a comment attribute on both strings to explain this duplication (case varies, but you can use android:inputType or android:capitalize in the presentation)
1 errors, 0 warnings
```

david-allison commented:
> 

Push please, I'll take a look

david-allison commented:
I apologize - I guess the lint runs with a `tools:ignore`, and the ignore just suppresses the output :/

BrayanDSO commented:
Is there anything actionable for me to do? This can sit if necessary, it's a low priority to me

david-allison commented:
Sorry, blocked on #https://github.com/ankidroid/Anki-Android/pull/11076

david-allison commented:
Should be fixed with a rebase onto `main`

BrayanDSO commented:
Here it works perfectly! Thank you very much for all the job

david-allison reviewed (approved):
Awesome!

mikehardy reviewed (approved):
Nice!
